### ISSUE TITLE
web-component-tester

### ISSUE THREAD

**Issue by pironim** [browserstack/automate-node-samples#7]
Guys how about example for web-components-tester - https://github.com/Polymer/web-component-tester

Sause labs support this framework. and looks like browserStack can be configured because it use selenium grid. but there is no examples

**Comment by nidhimj22** [comment]
@pironim  
Yes, that would be cool. Are you interested in writing an integration?

**Comment by tony19** [comment]
I've created a prototype [project](https://github.com/tony19-sandbox/wct-browserstack-demo) that demos WCT with BrowserStack. A WCT plugin would be a much cleaner integration, and I might try to tackle that next.

**Comment by daKmoR** [comment]
I just tested 
https://github.com/hotforfeature/wct-browserstack
and it works phenomenal :)
